simulation/src/xline_group.rs

```diff
@@ -126,6 +152,20 @@ impl XlineGroup {
             .unwrap()
             .1
     }
+
+    pub async fn crash(&mut self, id: ServerId) {
+        let handle = madsim::runtime::Handle::current();
+        handle.kill(id.to_string());
+        madsim::time::sleep(Duration::from_secs(2)).await;
```

Comment:
Is 2s enough? The time is simulated.

Reply:
Just copy from curp_group, changed to 10s